tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "outDir": "dist",
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "nodenext",
+    "moduleResolution": "node",
```

Comment:
I'm not going to lie, I have no idea what this means anymore

Reply:
I had to change this due to this TS error `JSX element type 'MuxPlayer' does not have any construct or call signatures.`. `nodenext` should be used nowadays according to TS docs but we might doing something wrong in Mux elements, not 100% sure.

https://www.typescriptlang.org/tsconfig#moduleResolution